qt/scientific_interfaces/Indirect/IndirectFitAnalysisTab.cpp

```diff
@@ -307,6 +284,13 @@ void IndirectFitAnalysisTab::setSpectrumSelectionView(
           m_fittingModel.get(), view);
 }
 
+void IndirectFitAnalysisTab::setOutputOptionsView(
+    IIndirectFitOutputOptionsView *view) {
+  m_outOptionsPresenter =
+      Mantid::Kernel::make_unique<IndirectFitOutputOptionsPresenter>(
+          new IndirectFitOutputOptionsModel(), view);
```

Comment:
Since this is created and then passed in immediately, it looks like its an owning pointer, is it possible to move in a smart pointer here? looking further in it looks like maybe its not due to QT